src/components/Asciinema.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+// Custom component to render the embedded Asciinema player.
+//
+// Usage:
+//
+// <asciinema id="267480"></asciinema>
+//
+
+class Asciinema extends Component {
+  constructor(props) {
+    super(props);
+    this.ref = React.createRef();
+  }
+
+  componentDidMount() {
+    const script = document.createElement('script');
+
+    script.setAttribute('id', 'asciicast-' + this.props.id);
+    script.setAttribute(
+      'src',
+      'https://asciinema.org/a/' + this.props.id + '.js'
```

Comment:
Could the `id` prop have some other name like `session-id` etc.? Mostly because the `id` attribute has a special meaning in HTML, and it would yield invalid HTML if the same `id` would appear more than once in a page.